src/index.html

```diff
@@ -1,10 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <title>React scaffold by ngParty!!!</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="shortcut icon" href="favicon.ico">
```

Comment:
this is not necessary and not covering all platforms as it's covered by http://realfavicongenerator.net/